.gitignore

```diff
@@ -0,0 +1,4 @@
+﻿.idea
```

Comment:
A .gitignore should only ignore project artifacts. Files that are specific to your editor can be ignored by editing `.git/info/exclude` on your laptop. You can also configure Git to ignore these files automatically when you create a new repo by adding a template.

Reply:
Thanks for the input 🙌